gpt_index/readers/notion.py

```diff
@@ -93,6 +93,22 @@ def read_page(self, page_id: str) -> str:
         """Read a page."""
         return self._read_block(page_id)
 
+    def query_database(self, database_id: str, query_dict: Dict[str, Any] = {}) -> List[str]:
+        """Get all the pages from a Notion database."""
+        
+        database_url = f"https://api.notion.com/v1/databases/{database_id}/query"
```

Comment:
could you move this to a global var at top of file? (consistent with other global vars)